### PR TITLE
Removes Freezer and Heater from Toxins on Box Station

### PR DESCRIPTION
**Pull request by Arthurdentist**
no heater or freezer

I remember a time when Toxins didn't get a roundstart heater or freezer. I think we should go back to such a time. 

As it stands you can blow the entire station with very little effort on Box without putting any real work in. This will mean that toxin bombers will have to put in at least a little bit of effort before they can destroy everything (even if its just going to the asteroid to fetch a freezer and burner from there).

### CODE REVIEW

**Comment by DamianX** [comment]
There is still a heater/freezer combo on the research outpost, this change would just add a 60 seconds delay.
I'm all for less bombs, but I don't think this is enough.

**Comment by Arthurdentist** [comment]
Not gonna stop people going further then this. Just a straightforward change that can be made relatively easily.

**Comment by BobdaBiscuit** [comment]
This changes nothing, you can have 6 devs constructed 4 minutes into the shift with the burn chamber alone. 
Fucking about with freezers and heaters actually slows down the process as they take time to cool/heat shit to the required temperature.

**Comment by ghost** [comment]
Remove toxins

**Comment by Kurfursten** [comment]
Move Telescience to Medbay

Telebombing is the real annoyance of getting bombed, and the only legitimate use of tsci is paramedic duty. Besides, Science is not hurting for toys at all and can still build stuff.

**Comment by ghost** [comment]
Yeah, telescience should be medbay/science instead of genetics beings medbay/science 
Or give paramedic a special one way teleporter

**Comment by ghost** [comment]
Someone(not me) should make a separate issue for telescience access

**Comment by SysadminJeroen** [comment]
@Kurfursten How about Telesci getting shared between Paramedics and Sci?

**Comment by Kurfursten** [comment]
There's no real reason science needs tsci. Calibrating it isn't difficult research. Science is tripping over themselves in toys - not to mention large artifacts, combat mechs, printable guns, bombs, gold core slime grenades, et cetera. I've never understood why science needs to be able to teleport their bombs, they have a bomb test range and a shuttle to travel to the roid. Other than "it has always been that way", I can't think of any reason scientists need it. Meanwhile we literally have a job about recovering bodies based on coordinates that is outclassed by the existence of this machine - and it's in a department with far less toys.

As an added bonus, if telescience was moved to the paramedic office, it would suddenly be in a prominent room with windows instead of a cult rape cave.

**Comment by ghost** [comment]
You don't need the freezer/heater to hit cap. Removing it just fucks with legitimate research by adding more chores that reduce the little time you have to actually test in the average hectic round without doing anything to stop shitters 6-bombing. I don't know what the hell you're thinking because it's really god fucking damned shitty when the round keeps fucking ending before or only a couple tests in while you're tweaking the next test to find a gain over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over and over. When they're punished constantly for even trying they end up stopping. There's already too much played all evening, got to do 4 tests the entire time. It's not worth even an inch further back towards that when it does nothing to stop antag bombings.

**Comment by ghost** [comment]
Science uses telesci for space exploration, recovering large amounts of space loot, recovering stuff the shitter teleporter flung out to nowhereville, telebombing blob/xenos/spiders and the usual recovery/blow and recover mmi, and in case of shitters stealing both hand tele's like usual they can stack and move large amounts of crates for whatever project as a ghetto hand-tele. If you're rustled about telebombings tell sec to grab people that ready a bunch of valves without testing next time.

Edit: @Jeroen52 just give paramedics general science access like mechanics and geneticists and they're able to use it just like those two. They really should have gloves and even then it's not hard for them to take a kelotane and find wires while they aren't lethal.

**Comment by FudgePucker** [comment]
Back when I played science I would just make a quick 6dev bomb mix in toxins, print a FOUR SHOT syringe gun, find and kill someone with maintenance access for their ID, plant bombs hidden in maintainance, and push the round ending button on your remote signaler. 
Freezers and heaters are the slowest way to hit bomb cap.

**Comment by Kurfursten** [comment]
There's no reason why any of that equates to science should have it. The department is bloated with toys it doesn't need.
* Explore Space: This is not the responsibility of scientists. Mechanics ("spacepod mechanics") or miners? Sure. Science is one of the only departments without EVA hardsuits, this is definitely not their job.
* Recovering loot from space:  This is firmly a miner/paramedic job. Why is science suited for this other than "they are the ones with telescience"? Performing this labor doesn't require you be in a department with a protolathe or bomb lab.
* Recover stuff from teleporter: Same as above.
* To move crates (Cargo): A very strong reason to remove it from science. Cargo needs to set up telepads if they want to teleport crate, there's reason scientists need crate teleportation unlike every other department at roundstart. Even then, unlike any other department research COULD build their own for autism projects if they wanted to. Moreover, unlike crate-heavy departments like engineering and botany, science tends to request less crates than any other department.  This is a really confusing point. Even medbay wants more crates than science.
* To kill Xenos/Blob/Spiders: This is, again, about making another department (security) irrelevant. Why can't security deliver the bomb? Why can't you get medbay to send the bomb if this is the most cooperation-oriented game mode? We don't place rechargers in science so that scientists can go in guns blazing on blob, they need to cooperate with security.

Remember, scientists also have access to latex gloves plus an autolathe, so the opposite argument is also true. They are actually even BETTER oriented than a paramedic to hack into a room if they want access. If the scientist in question is still rustled that they can't explore space as a role that doesn't even have maintenance access, and they're not willing to hack, in they can always ask the HOP for access.

**Comment by ghost** [comment]
may want to remove the freezer/heater from deff toxins, too.

**Comment by ghost** [comment]
Because outside of 5 minutes of mashing a button, removing the rtable and windoor on rnd, and putting a cell into the borgs their job is whatever and whatever often translates into gitgudder at something new time.

* Psst. I wonder if it's because the science themed hardsuits are xenoarch suits which they don't need in eva. Could that possibly be the reason???????? Mining has no more of a claim on space exploration than science and spacepods (a scientist duplicate no less) can also only loot what they can fit in their backpack and 2 toolboxes unless they telesci loot back like the filthy dupe they are. As a general mission to discover shit science is well within bounds to explore. They have a space suit in telescience for space exploration regardless. 

* A miners job is to mine. If you want to stretch it for space exploration it fits science just as much. Paramedic has only the loosest claim through corpses with sensors but still wont ask any of us to zap [coords] back. A miner should be mining. A paramedic should be readying his crash kit, recovering people, and camping sensors. Science has loose goals that leaves them free enough to focus on other things instead.

* No, it really, really ,really fucking shouldn't because in the real world. In the world of actual gameplay things break down. It's not the perfect fantasy land where everyone is competent and alive. Often enough you have to roll up your sleeves and make do with the absolute dogshit reality of how terrible people and rounds can be if you want to build your super autism SME gorillion emitter platform this century so sometimes a girder and your eng gps are the answer. Even when it's just some misc research project that can't be easily mailed to it's an option for when cargo yet again is a horrific mess instead of nice mr rcs man saving the day.

* Gee, I dunno, maybe that intelligent mobs aren't going to stick around while the setter has to bumble through doors and firelocks to get clear? That they can't rapidly carpet bomb a say hydro maint shitbox of resin and xenos? Maybe that you have to rapidly offset double bomb to catch the ones in the heavy radius in even a full blown viro outbreak? Maybe that you can't just phase through blob walls to clear the full range you could otherwise to cut a tunnel or blast a buffer zone of station away? Medbay is less likely and able to get camera consoles setup for spotting your strikes and if you want to talk cooperation, while sec is busy having to order and unbox cargo guns themselves because of the cooperation meme you'll later telebomb near a core then tele a locker of 2-3 guys with 2 backpacks of guns each to snipe the core. A paramedic is far less likely to have learned to be able to accurately hit a target without coords because they never had a reason to because they have a real job they should be doing.

Paramedics also have access to latex gloves and multitools in addition to being able to just grab a kelotane out of med storage for the first shock. If you haven't played in so long you think the get access roulette is a practical answer to anything I'm speechless. There's a damn good reason why things are as shit as they are. P.S. Space has fucking nothing to do with maint you ass.

**Comment by eplgr** [comment]
:-1: 
I added those for a reason. Removing them is a really poor design choice. The only thing you are achieving by doing this is annoying the players who want to make bombs or research toxins.

If you think toxins overall should not be able to do what it currently does, then you should aim to rework how toxins/gases work, instead of trying to remove these tools which are not the root of what you consider to be a problem.

**Comment by eplgr** [comment]
I suggest giving #14027 a quick read, just `ctrl`+`f` and search for `heater` or `freezer`.
No one was bothered by the changes and the only comments about this are positive ones.

**Comment by Arthurdentist** [comment]
I hardly think toxin scientists are that inconvenienced by this change, but it does mean they would have to do a little more travelling then they do currently. Simply put I don't think its the massive inconvenience you are making it out to be and all this change might really do is slow down people who want their 20 minute low-pop bomb round. 

I can't actually remember the last time I saw a traitor scientist use the mixing chamber to make a bomb either. They're easier to catch when they do that because an AI or someone walking down the science corridor can see the lights emanating from the chamber. That might just be me though, and I could very well be missing it. 

The other group this PR inconveniences will be 'legitimate' toxin players, but then almost any change would necessarily impact those players anyway, certainly tweaking the formulas involved would. Would you be happy with a PR that drastically increased the difficulty of making an optimum bomb? What kind of balance changes do you think would be more acceptable or are we fine as is?

**Comment by eplgr** [comment]
It's true, no one has been using the mixing chamber lately. That's due to the fact that people realized and learned that it's way more effective to use heaters/freezers, this started happening before my PR and has nothing to do with what you are talking about. This will not change regardless of the outcome of this PR.

In a few words, aiming to make toxins less functional as a way to "nerf" the bombs themselves is just a shit change.

**Comment by eplgr** [comment]
>Would you be happy with a PR that drastically increased the difficulty of making an optimum bomb? What kind of balance changes do you think would be more acceptable or are we fine as is?

I don't care, you are the one who is annoyed by this and should come up with a proper solution.

If you want to nerf bombs, go nerf bombs. Leave the poor tools alone.

**Comment by Arthurdentist** [comment]
Im not sure how this makes toxins 'less functional' considering it has no relation to the way bombs are made or their effects. I would have thought that lowering the bomb cap for example or making it harder to get an optimal mix would have a much greater impact on toxins 'functionality'. We could start the round with everything hooked up from the get go, arguably that'd increase the functionality massively. Why drag canisters across hallways when we can put them all there, hooked up and ready to go?

**Comment by eplgr** [comment]
There's a difference between, for example, having to set up the singularity, the SMES and so on, which obviously lead to a gameplay we desire, with a very basic way to set them up, and then more optimal and better ways to do it which are rewarding and require higher skill-knowledge.

Then there's just missing stuff from your departament that are basically randomly placed in the opposite corner of the station and you won't want to/can't start to do your actual job because you have to go pick them up, with ALSO no departement interaction at all (which we also want). This is not fun and this does not add anything positive to the game. This is just annoying.

You want to nerf bombs? Then: nerf them. That's my suggestion.

**Comment by Arthurdentist** [comment]
Well you could lock them behind Atmospherics to encourage department interaction, which we also want. You'd retain your basic method of making bombs, and you'd have a route to more optimal methods, but I think you'd appreciate that change a great deal less.

 I'm not actually smart enough to approach something as significant as changing toxins in a substantial way. I just want to slow down the speed a bit at which a guy can make a perfect bomb, and I think doing so might make for slightly better rounds.

**Comment by eplgr** [comment]
Atmos already has a freezer+heater that are never used, and it is well-known that atmos is way better at toxins, than toxins. My PR kinda aimed to shorten that breach. This is literally why I added a pipe dispenser, so you don't have to rely on people playing atmos/engineering and losing tons of time waiting for them and other shit that is already discussed in my PR. They are not cargo or a job that really wants to do that.

I think I've said enough, so let's see what happens.

**Comment by mrpain666** [comment]
I dont see the point of this PR.

You're just making a max dev take 10 minutes to make instead of 6.

If you really do want toxins/science nerfed, as others have stated, either rework gasses (have fun with that one!) or move/remove telescience.

**Comment by ghost** [comment]
Holy jesus cock fucking shit arthur, if you don't know how to make a cap bomb out of just burn chamber gas and the stock plasoxy gas you shouldn't be fucking with the heater and freezers. THEY. DO. NOT. NEED. HEATERS. OR. FREEZERS. TO. GET. BOMBCAP. USING. HEATERS. OR. FREEZERS. ACTUALLY. TAKES. THEM. LONGER. DUE. TO. HOW. SLOW. THEY. ARE. FULL. FUCKING. STOP. YOU. DON'T. KNOW. WHAT. YOU. ARE. DOING. IT. DOES. NOT. STOP. OR. EVEN. SLOW. 6-BOMBERS. You can crank the cap all the way down to 1,3,6 you're still not going to stop 6-bombers. If you want to "help", go after greytiders that sax all over the place being actually problematic shits wasting sec's time on petty bullshit instead of things being calm enough that they can actually have a spare guy on cams to notice hey, bomb guy's just rigged up the 6th bomb without testing and now he's fiddling with telescience, he got weird access earlier, or heading out of the dept maybe we might have a problem here because that's the only thing that's going to reign it in.

Heaters and freezers give consistent and precise results which are better for when you begin experimenting on expanding your theoretical yield autism once you've finished up on dirty co2 bombs from the burn chamber. They help you experiment with flamethrower compositions. They help you with the critical task of precision in your primer tank research for multi-step bombs. They're very helpful on discovering consistent canister bomb recipes instead of a castrated version that focused on being able to explode from a wide range of primers instead of efficiency due to inconsistencies in your gasses through rounds. They help you research into one-tank bombs where they help you sidestep needing to valved heat exchanger the chamber output then use a connector-pump-connector to ghetto heater your gasses to where you wanted them because burn chamber gas is too hot, too dirty, and too inconsistent to rely on when a minor difference spoils the tank or causes it to explode during mixing. So when they have to take a little detour to go bumpbump it onto the shuttle and back through all those doors that's that much less time they have to actually test. With how absolute garbage most rounds are it's easy to simply not have time to test or you wont be able to adjust the mix for subsequent tests because shitters fucked everything and instead of solving it they're just gonna shuttle. So then next round you're spending all that time retracing your steps and it's gonna end early again. and it's gonna end early again.and it's gonna end early again.and it's gonna end early again.and it's gonna end early again.and it's gonna end early again.and it's gonna end early again.and it's gonna end early again.and it's gonna end early again.and it's gonna end early again.and it's gonna end early again.and it's gonna end early again.and it's gonna end early again.and it's gonna end early again. Which gets really goddamn fucking annoying when you're trying to research a new idea.

> We could start the round with everything hooked up from the get go, arguably that'd increase the functionality massively. Why drag canisters across hallways when we can put them all there, hooked up and ready to go?

I dunno, probably the part where there's numerous decisions you need to make that about what gas composition you use and how you do it that impact yield, time needed before you can use it, what you can make from it, and ability to modify it further if tweaks are needed without needing a socket/water-potass or having to have repiped it into waste before. The interaction meme is fucking garbage cause you know what actually fucking happens? Half the time atmos doesn't listen, the AI is one of the chucklefucks so you end up just thermiting it out which as expected leads to all the issues that stem from that to the point that time-wise you average out better just researching and making your own from scratch which yet again is another intent didn't go as planned situation. 

tl;dr: In case you didn't know, the big bad 6-bomb boogeyman already hit cap without the freezer/heater. He does it quick. He does it dirty. His mix doesn't care about heaters or freezers and this isn't going to stop it. Don't beat up legitimate research without even accomplishing the intended goal.

**Comment by SonixApache** [comment]
I for one agree with SarahJ.
This PR is pointless.
A punishment for those doing actual research, while not affecting the exact people it's aimed at.

If you want to make bombing less effective, think of a way to make bomb making take more *actual* effort, like changing the formulas. Or make bombing harder by locking away the TTVs. Or make explosions have less effect on reinforced walls. Or whatever.

This isn't the way.

**Comment by ghost** [comment]
>4 paragraphs
Tl;dr

**Comment by ghost** [comment]
If you really want to fix dudebombslmao, make a big motivational poster showing how to:

* Mass produce diona
* Mass produce Golems
* Mass produce Posibrains
* How to load a posibrain into a mech and installing an electropack so you can eject it if needed.
* Mass produce pAI's and slot them into bots or just have them be helpful
* How to card and use the AI integrity restorers
* Constructing AI's in general
* How to order the NT tech crate and make MaMMI's to put posibrains in a human body
* How to humanize a monkey, debrain, implant a diona brain or MaMMI into the human body and defib
* How to use the ATM to buy all the metal from cargo then mass produce borg/mommi shells
* How to run the borer pen easier and flush more eggs ordered from cargo
* How to construct your first clamp-rcd-cargo bay of modules/supplies Repair Firefighter Ripley, weld to repair damage, and how to change its cells
* Basic mech pilot guide
* Deconstructing rwalls and moving one of the spare oxygen miners for dummies
* Constructing a cargo AME
* How to turn on and swap an air conditioner cell
* Retards first oxygen compressor
* Retards first freezer bank air conditioner system
* Diagram for a windoor-tripped timing airlock entrance that handles fast repressurization and sidesteps the servers continued inability to close firelocks and airlocks behind them instead of zasing the safe zone with blasmafire or vacuum
* Diagram for basic healing chems and dosages to operate in hazardous conditions without an eva suit
* A reminder that repairs don't have to look nice, they have to work. Just glass off most of the breach and rebuild the lost machinery elsewhere
* A reminder to heads to actually swipe red, how to do it, and for captains and ghosts to actually use the now automated Call-ERT function.
* A reminder to stay in your body or at least keep tabs on it if you're dead
* A reminder to get back in your body when you logout so we can clone you and mark you off the list
If you want to go the extra mile, a quickref sheet of construction/deconstruction steps, parts subtotals, and parts totals for a full core machinery in a room set for quickly getting people up to speed instead of them bumbling through

We have the means to easily bounce back from 6-bombs. We have the means to bring unlimited dead back into human bodies. What's lacking is people that know how and people lacking the willpower to respond and stake a claim on something that needs doing when it's called out instead of just taking the easy antag reroll. With a handful of guys that are willing to work and respond to and claim task assignments from the station announcement you can have a new dept with the core machinery pressurized, powered indefinitely, and cloning/sucking ghosts in ~20 minutes as people pitch in.


@SonixApache Worst comes to worst, they can just use delayed canister bombs or regular canister bombs to serial bomb the station to a standstill. And that's not even getting into assembly lining a large supply of one-tank bombs or one-tank firebombs and seeding them everywhere with prox or the other to turn the station into a minefield and killing off most crew worth a damn. If they do go nuclear I'll still miss nocap blast cannons sniping xenos and spider queens. ;_;7

**Comment by Arthurdentist** [comment]
@SarahJohnson489 After reading your comment I'm actually inclined to agree, this isn't the best way to go about this and it doesn't fundamentally solve the issue. The issue is that toxins is an autistic mess thats only suitable for early shuttle calls or theoretical numbers on sheets of paper, and creating extra steps wont actually solve that. The entire role is fundamentally flawed. As far as I see it, toxins only really has two, arguably "useful" functions. 

1.) Helping miners, a function that isn't particularly useful given that miners have free access to significant explosions already (Gibtonite), without involving scientists
2.) Blowing up blobs. I'm actually surprised that this one doesn't happen more often, because its a very effective method of killing them. Its pretty much the only antag type that can reliably be killed with bombs without having admins breathing down your neck for pretty obvious reasons. You could probably add blowing Malf AIs to that, but that is tricky business given that the AI core is teleblocked. 

Anything else boils down to theorycrafting or antagging. Toxins scientist is a role that is about as useful as the librarian but with an infinitely greater capacity to ruin a round. 

I do think you are massively overreacting with your fuck off huge paragraphs of righteous indignation and I hope you realise that my intentions didn't come from a vindictive desire to make your life miserable or something. 

I'm going to close the PR and I hope someone will address the issue of easy antag bombings better then I can. I don't think Toxins should be removed or necessarily nerfed, I just don't like having back to back 20 minute bombing rounds every round. 

**edit** I also don't think "WE STAY!" is an actual solution to the problems caused by toxins. Trying to fix the broken station five times in a row is about as fun as pulling teeth, especially for people who, like yourself were trying to test or do other things.

**Comment by SonixApache** [comment]
the obvious answer is to remove toxins

*with a bomb*

**Comment by ghost** [comment]
Yeah, you pretty much hit the nail on the head. IMO, toxins has a couple places to shine but unfortunately, they're mostly a dingdong risk, rely on bus/random events for their preferred targets, or rely on bus to get the cap removed to carpet bomb the infested maint/viro fast enough before the xenos run out and hostile mobs slip into space.

That said, its main contribution to the greater good would be getting a couple blast cannons and locker of flamethrowers + refuel can to sec or at least each other. Even if the bombcap isn't lifted you can still go on a raid against xenos/spiders with something solid to shoot through after you've lost the ability to easily snipe a xenos within dev range through an rwall if you can get someone to be your camera spotter or diagonally from cover if they're baiting you to run in. With a single shot you can gib a spider queen (500 hp in one go instead of 17 laser hits!)/xenos/Ops/rogue mech (squids usually but lord almighty when you come around the maint corner and they're lined up)/rogue borg/multi-shot into the rogue/malf AI core to assassinate it and then in the case of the mech your battle buddy can take out the pilot.

Tighter flamethrower mixes, so they don't have to spend as much time in the flames to get heavily slowed or pain crit if chased, are amazing in close quarters where they can't get clear and if you sacrifice fuel, can work as a ghetto ricochet laser/ricochet lawgiver to force people out of cover with the cryo room sized plume. I hate how they no longer burn weeds but you'll convert the first time you roll up to a nest and roast them in their own tunnels without risking the neurotox snags of a sustained laser barrage. Absolutely love when you find a good scientist when you poll them with pda's and later drops off a locker of flamers and an atmos analyzer at sec. I always miss the hive away missions but the couple times the hive vault hits the station the flamers have been damn effective at making pushes and would dominate if I could ever get people to supply chain the reloads.

One-tank bombs, while grenades would be preferred, are easily mass produced once mix is prepped and are pretty effective in total war scenarios like cult, malf, subv, and rev, give you a way to get through rwalls like the ai core or a hardened cargonia/blast doors like a science that hits biohazard shutters then starts sensor abductions faster, quickly gib a flashed rogue borg, necromorph or room of flamer'd zombies from a zombie/necro staff, drop an armed one to take out someone trying to catch up and pass your cover to laser you, or even simply breach distro to cut off the flood. Much like the blast cannon, you can use them to gib spiders, xenos and other hostile mobs depending on your settings but with less collateral. Socking a bear trap away in your flex backpack slot is nice but the one-tanks have some utility over them.

One-tank firebombs allow for rapid clearing of cyanide/etc. vines, spiders, or xenos/weeds/eggs then engineering or you can come in with foam extinguishers or breach with a one-tank bomb then patch and repressurize. Like flamers, in a serious pinch they will force others out of cover without exposing yourself.

Delayed canister bombs give you a large supply of stronger breaching charges to clear a path that once setup you can get anyone to assembly line primers and run beacons or deliver themselves. Bless the crews merry little heart when they help the eng borg assemble them, scan and blueprint the teleporter, and spread telebeacons. Your heart is almost as warm as the burn chamber they came from once you crowbar that flatpack for a teleporter. People exclaiming what the fuck while the explosions keep going while you purge a spider nest is a close second though.

As a last resort method, backing up and sending anyone left alive into xenos nests with canister bombs does a pretty hefty blow to them if you can get them to actually listen and stage the blitz so there isn't time to weed back up. The few times we've gotten the cap lifted it was an absolute shoah of dying xenomorph screams though as the 14 devs rocked the nests and all of them in it to the ground. I've taken out large chunks of bus ops by greytiding into toxins and whipping up a cap can out of the researchers leftovers a couple times. Don't need as much combat skill to just be on the same screen for 2 seconds and it's a darn shame tox storage isn't reloaded with em while they're doing their other research as they're damn effective.

For blobbers, people are pretty poor at listening, even when you're doing it as the RD, and therefore can make the get clear or be queer announcement yourself, people don't listen so while toxin quick mix with the cap lifted, or your tox storage of 6-bombs and canister bombs for event blobs, is the single strongest tool on the station to cripple a blob's advance as it has to spend 8+ times as much resources to advance a single tile of space it's simply inevitable that you're going to gib or crit 3-7 people if you carpet bomb a DMZ of space tiles to mount a resistance over. Then since it's usually pretty hard to get a raiding party together you end up doing a solo raid to bomb and laser the ones that are near space. Silicon assistance also enables sustained carpet bombing with canister bombs after the valves are gone as fast as the rubes you recruited can bring more metal and assembly line primer tanks to send out for the silicons to detonate letting attempts to expand around the front be curbed.

Don't worry, I didn't feel that way, it's just a knee jerk when it looks like another raid on toxins is coming. Some day I dream that we'll get engineering and reviving knowledge widespread enough in the meta that people stop running scared of a simple bombing or loose goose and with it, that shittagging will die off

**Comment by FudgePucker** [comment]
You can still bomb the AI without telebombs. You just need two bombs.

**Comment by PunishedHoratio** [comment]
This is bullying and bullying is wrong.  

Ultimately however I am ambivalent towards this change.  If I feel like bombing, I'll just go grab them from the outpost or make my own (which I used to do anyways).  As someone else said, you can use the mix if you REALLY want to speed run it (but I prefer my 9-10dev autism even if it costs me a few minutes compared to using a mix).

You can't stop the rhumba beat.
